### PR TITLE
First set of changes for settings pages new design

### PR DESCRIPTION
**Pull request by bexsoft**
## What does this do?

Implements the first page for new settings page design.

## How does it look?

Screen Shot 2020-12-08 at 19 58 24
Screen Shot 2020-12-08 at 19 58 08
Screen Shot 2020-12-08 at 19 58 00

### CODE REVIEW

File: portal-ui/src/screens/Console/Configurations/ConfigurationPanels/ConfigurationsList.tsx [hunk 1]
```diff
@@ -73,8 +116,9 @@ const ConfigurationsList = ({ classes }: IListConfiguration) => {
           // We redirect Browser
           history.push(url);
         } else {
+          setCurrentConfiguration(1);
           setSelectedConfiguration(element);
-          setEditScreenOpen(true);
+          //setEditScreenOpen(true);
```
Comment (Alevsk) [inline portal-ui/src/screens/Console/Configurations/ConfigurationPanels/ConfigurationsList.tsx:121]: can this be deleted?

**Review by Alevsk — CHANGES_REQUESTED** [review]

**Comment by cesnietor** [comment]
Works fine I just saw that it is alligned a little bit lower than other pages.
Screen Shot 2020-12-09 at 12 02 00 PM
Screen Shot 2020-12-09 at 12 02 08 PM

File: portal-ui/src/screens/Console/Configurations/ConfigurationPanels/ConfigurationsList.tsx (hunk 1 above)
Comment (bexsoft) [reply portal-ui/src/screens/Console/Configurations/ConfigurationPanels/ConfigurationsList.tsx]: done

**Comment by bexsoft** [comment]
> Works fine I just saw that it is aligned a little bit lower than other pages.
> Screen Shot 2020-12-09 at 12 02 00 PM
> Screen Shot 2020-12-09 at 12 02 08 PM

Let me check, what browser are you using? I see these titles aligned:

Screen Shot 2020-12-09 at 13 30 52
Screen Shot 2020-12-09 at 13 30 45

**Comment by cesnietor** [comment]
@bexsoft  works fine, I had to make the assets

**Review by cesnietor — DISMISSED** [review]

**Review by Alevsk — APPROVED** [review]
LGTM

**Review by dvaldivia — APPROVED** [review]

**Review by Alevsk — APPROVED** [review]

**Pull request merged**